CSharp/ArticleProjects/ApplicationInsights/README.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Monitoring and debugging Azure Batch applications with Application Insights - Azure | Microsoft Docs
```

Comment:
Does this stuff show up nicely in a rendered markdown on GitHub?

Reply:
It seems like that's a no, it doesn't -- can we make it look pretty?

Reply:
Removed table